File: java/src/test/java/org/mydotey/quantileestimator/kll/ClassicQuantileEstimatorTest.java
package org.mydotey.quantileestimator.kll;

import org.mydotey.quantileestimator.facade.QuantileEstimators;
import org.mydotey.quantileestimator.value.LongCalculator;
import org.mydotey.quantileestimator.value.LongComparator;

/**
 * @author koqizhao
 *
 * Apr 1, 2018
 */
public class ClassicQuantileEstimatorTest extends QuantileEstimatorTest {

    public ClassicQuantileEstimatorTest() {
        _estimators.put(Long.class,
                QuantileEstimators.newClassicEstimator(LongComparator.DEFAULT, LongCalculator.DEFAULT));
    }

}

File: dotnet/test/MyDotey.QuantileEstimator.Tests/Classic/ClassicQuantileEstimatorTest.cs
using System;

using MyDotey.Quantile.Value;
using MyDotey.Quantile.Facade;

/**
 * @author koqizhao
 *
 * Apr 1, 2018
 */
namespace MyDotey.Quantile.Tests
{
    public class ClassicQuantileEstimatorTest : QuantileEstimatorTest
    {
        protected override void AddQuantileEsimators()
        {
            _estimators[typeof(long)] =
                    QuantileEstimators.NewClassicEstimator<long>(LongComparer.DEFAULT, LongCalculator.DEFAULT);
            _estimators[typeof(int)] = QuantileEstimators.NewClassicEstimator<int>(IntComparer.DEFAULT, IntCalculator.DEFAULT);
        }
    }
}